content/digital-service-standard/11-measure-performance/guidance.md

```diff
@@ -2,4 +2,4 @@
 layout: text/textblock
 ---
 
-[Measuring service performance](https://www.dta.gov.au/standard/measuring-performance/)
+[Measure success](/content-strategy/set-goals-measure-success/measure-success/)
```

Comment:
This page has a different focus from the original link. We'd need to find something more appropriate.

Reply:
I'm checking with the Insights team on this one.

Reply:
@sirleech thanks ok after talking to the Insights team can we delete this partial and the `Guidance related to this criteria` heading partial (as there aren't any other guidance links) Happy for you to do this or us to pull this PR and update on our end.